### PR TITLE
[DO NOT MERGE] init cron pkg no matter what

### PR DESCRIPTION
**Pull request by ruseinov**
As requested - force init cron package migration for investigation.

DO NOT MERGE - this change is going to be discarded after the investigation.

### CODE REVIEW

File: migration/init.go [hunk 1]
```diff
@@ -58,5 +58,7 @@ func (Initializer) FromGenesis(opts weave.Options, params weave.GenesisParams, k
 		}
 	}
 
+	MustInitPkg(kv, "cron")
```
Comment (husio) [inline migration/init.go]: I think this is bad approach. Initialization should be done in the genesis file.
This creates a dependency and hides a single package initialization in unrelated code.

**Comment by iov-bot-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/iov-one/weave/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: migration/init.go (hunk 1 above)
Comment (ruseinov) [reply migration/init.go]: @husio we are fixing this one off, please read a Slack message. It's not for master branch of production use other than extracting data from the testnet.

Comment (husio) [reply migration/init.go]: I am confused. I thought you want me to review it. I don't think this is not something that can get into `master`.
